React/Base/macOS/RCTUIKit.m

```diff
@@ -609,3 +609,21 @@ - (instancetype)initWithFrame:(NSRect)frameRect
 }
 
 @end  // ]TODO(macOS GH#774)
+
+@implementation RCTUISwitch
+
+- (void)setOn:(BOOL)on animated:(BOOL)animated {
+	self.state = on ? NSControlStateValueOn : NSControlStateValueOff;
+}
+
+- (BOOL)on
```

Comment:
Getter should be `isOn`, not `on`, as per `UISwitch` header:

```swift
@property(nonatomic,getter=isOn) BOOL on;
```

Reply:
Good catch! Fixed!